packages/runner-ct/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@packages/runner-ct",
+  "version": "0.0.0-development",
+  "private": true,
+  "main": "index.js",
+  "browser": "src/index.js",
+  "scripts": {
+    "build": "webpack",
+    "build-prod": "cross-env NODE_ENV=production yarn build",
+    "clean-deps": "rm -rf node_modules",
+    "postinstall": "echo '@packages/runner needs: yarn build'",
+    "test-unit": "ts-mocha --config ./test/.mocharc.js",
+    "watch": "webpack --watch --progress"
+  },
+  "dependencies": {
```

Comment:
I'd like to go through and audit these dependencies to make sure we're using them. Additionally, I'm a little nervous about the version ranges. I don't want to conflict with Cypress. Can we just go through and ensure we're pinning correctly?